node-notes-cli/app.js

```diff
@@ -0,0 +1,75 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+// Reading data from data.json
+async function readNotes() {
+  const jsonData = await readFile('./data.json', 'utf8');
+  const data = JSON.parse(jsonData);
```

Comment:
You repeat these 2 lines of code several times. It would make maintenance easier if you put them into a separate function.